docs/docs/contributing/development.md

```diff
@@ -32,10 +32,10 @@ multipass mount $HOME/projects/hestiacp hestia-dev:/home/ubuntu/hestiacp
    git clone https://github.com/YourUsername/hestiacp.git $HOME/projects
    ```
 
-1. Create an Ubuntu VM with at least 2G of memory and 15G of disk space
+1. Create an Ubuntu VM with at least 12GB of memory and 15GB of disk space
```

Comment:
We should add a note as  it depends on if PHP / Nginx need to be build or not. 99 % of the cases it is not needed only now for 1.9.0